admin/base/settings/defaults.py

```diff
@@ -31,10 +31,14 @@
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
+    'django.contrib.messages',
+    'django.contrib.sessions',
     'django.contrib.staticfiles',
 
     # 3rd party
+    'admin.common_auth',
```

Comment:
Does admin auth need to be in here? I don't load the spam app in https://github.com/CenterForOpenScience/osf.io/pull/4566 but it still works.